RubberduckTests/Refactoring/RemoveParametersTests.cs

```diff
@@ -349,5 +818,29 @@ private static Mock<IRefactoringPresenterFactory<IRemoveParametersPresenter>> Se
             factory.Setup(f => f.Create()).Returns(presenter.Object);
             return factory;
         }
+
+        private void SetupProject(string inputCode)
```

Comment:
This should be throwing a warning about hiding base class methods. Is this intended? Why not use the base implementation?